e2e/capacity/change_staking_target.test.ts

```diff
@@ -1,54 +1,41 @@
-  stakeToProvider, CHAIN_ENVIRONMENT,
-  TEST_EPOCH_LENGTH, setEpochLength,
+  createKeys, createMsaAndProvider,
+  stakeToProvider,
   CENTS, DOLLARS, createAndFundKeypair, createProviderKeysAndId
-}
-  from "../scaffolding/helpers";
-import { firstValueFrom } from "rxjs";
-import { MessageSourceId} from "@frequency-chain/api-augment/interfaces";
+} from "../scaffolding/helpers";
 
-describe("change_staking_target tests", () => {
+const fundingSource = getFundingSource('capacity-replenishment');
+
+describe("Capacity: change_staking_target", function() {
   const tokenMinStake: bigint = 1n * CENTS;
   const capacityMin: bigint = tokenMinStake / 50n;
 
-  const unusedMsaId = async () => {
-    const maxMsaId = (await ExtrinsicHelper.getCurrentMsaIdentifierMaximum()).toNumber();
-    return maxMsaId + 99;
-  }
-
-  before(async () => {
-    if (process.env.CHAIN_ENVIRONMENT === CHAIN_ENVIRONMENT.DEVELOPMENT) {
-      await setEpochLength(devAccounts[0].keys, TEST_EPOCH_LENGTH);
-    }
-  });
-
-  it("happy path succeeds", async () => {
+  it("happy path succeeds", async function() {
       const providerBalance = 2n * DOLLARS;
       const stakeKeys = createKeys("staker");
-      const oldProvider = await createMsaAndProvider(stakeKeys, "Provider1", providerBalance);
-      const [_unused, newProvider] = await createProviderKeysAndId();
+      const oldProvider = await createMsaAndProvider(fundingSource, stakeKeys, "Provider1", providerBalance);
+      const [_bar, newProvider] = await createProviderKeysAndId(fundingSource, providerBalance);
 
-      await assert.doesNotReject(stakeToProvider(stakeKeys, oldProvider, tokenMinStake*3n));
+      await assert.doesNotReject(stakeToProvider(fundingSource, stakeKeys, oldProvider, tokenMinStake*3n));
 
       const call = ExtrinsicHelper.changeStakingTarget(stakeKeys, oldProvider, newProvider, tokenMinStake);
-      const [events] = await call.signAndSend();
+      const events = await call.signAndSend();
       assert.notEqual(events, undefined);
   });
 
   // not intended to be exhaustive, just check one error case
-  it("fails if 'to' is not a Provider", async () => {
+  it("fails if 'to' is not a Provider", async function() {
     const providerBalance = 2n * DOLLARS;
     const stakeKeys = createKeys("staker");
-    const notAProvider = await unusedMsaId();
-    const oldProvider = await createMsaAndProvider(stakeKeys, "Provider1", providerBalance);
-    await assert.doesNotReject(stakeToProvider(stakeKeys, oldProvider, tokenMinStake*3n));
-    const call = ExtrinsicHelper.changeStakingTarget(stakeKeys, oldProvider, notAProvider, tokenMinStake);
-    await assert.rejects(call.signAndSend(), {name: "InvalidTarget"})
+    const oldProvider = await createMsaAndProvider(fundingSource, stakeKeys, "Provider2", providerBalance);
+
+    await assert.doesNotReject(stakeToProvider(fundingSource, stakeKeys, oldProvider, tokenMinStake*3n));
+    // const notAProvider = 3;
```

Comment:
nit: Should these comment get removed or what is the purpose of keeping them?

Reply:
Thank you.  This one I had commented out in debugging a bunch of tests and it needed to be put back in.